roaring/src/bitmap/serialization.rs

```diff
@@ -47,6 +48,143 @@ impl RoaringBitmap {
         8 + container_sizes
     }
 
+    /// Creates a `RoaringBitmap` from a byte slice, interpreting the bytes as a bitmap with a specified offset.
+    ///
+    /// # Arguments
+    ///
+    /// - `offset: u32` - The starting position in the bitmap where the byte slice will be applied, specified in bits.
+    ///                   This means that if `offset` is `n`, the first byte in the slice will correspond to the `n`th bit(0-indexed) in the bitmap.
+    ///                   Must be a multiple of 8.
+    /// - `bytes: &[u8]` - The byte slice containing the bitmap data. The bytes are interpreted in little-endian order.
```

Comment:
`endianness` doesn't apply within a byte, endianness refers to the order of bytes within a word.

The `bitvec` crate refers to this order as "[Least-Significant-First](https://docs.rs/bitvec/latest/bitvec/order/struct.Lsb0.html)" bit order, and I think that's a more accurate description.